### ISSUE TITLE
ML models selection

### ISSUE THREAD

**Issue by jdomdev** [Bootcamp-IA-P4/ai-decission-tress-for-regression#12]
Which **models** can best **fit** our case study.

**Comment by jdomdev** [comment]
Possible models to test:
 
- Start with Random Forest as a baseline - will give you a quick understanding of the problem
- Optimize XGBoost for your main solution - better accuracy/time ratio
- Test the neural network if you observe complex patterns in the residuals of the other models.
